src/components/Profile/Profile.scss

```diff
@@ -0,0 +1,56 @@
+@import "~styles/main";
+
+//global styles across all profile elements
+.profile {
+  font-family: Inter;
+  .button {
+    margin: auto 2px;
+  }
+  .profile__title {
+    margin: auto 20px auto 0px;
+    color: color(pure-black);
+    text-transform: capitalize;
+    font-size: 1.125rem;
+  }
+}
+
+.profileTop {
+  margin: 10px 20px;
+  width: 100%;
+  display: flex;
+  align-items: row;
```

Comment:
Is there a `row` value to align-items property?

Reply:
It was a mistake. I have resolved